scipy/optimize/_dual_annealing.py

```diff
@@ -637,11 +637,16 @@ def dual_annealing(func, bounds, args=(), maxiter=1000,
     # Strategy chain instance
     strategy_chain = StrategyChain(accept, visit_dist, func_wrapper,
                                minimizer_wrapper, rand_state, energy_state)
-    # Run the search loop
     need_to_stop = False
     iteration = 0
     message = []
+    # OptimizeResult object to be returned
+    optimize_res = OptimizeResult()
+    optimize_res.sucess = True
```

Comment:
it's spelled with two `c`'s.

Reply:
Fixed and added asserts in existing tests for this.